PatternPal/PatternPal/Services/RecognizerService.cs

```diff
@@ -146,25 +130,38 @@ public class RecognizerService : Protos.RecognizerService.RecognizerServiceBase
     }
 
     /// <summary>
-    /// Converts an <see cref="Recognizers.Abstractions.ICheckResult"/> to a <see cref="CheckResult"/>, which can be sent
+    /// Converts an <see cref="ICheckResult"/> to a <see cref="CheckResult"/>, which can be sent
     /// over the wire.
     /// </summary>
-    /// <param name="checkResult">The <see cref="Recognizers.Abstractions.ICheckResult"/> to convert.</param>
-    /// <returns>The <see cref="CheckResult"/> instance created from the given <see cref="Recognizers.Abstractions.ICheckResult"/>.</returns>
-    private CheckResult CreateCheckResult(
-        Recognizers.Abstractions.ICheckResult checkResult)
+    /// <param name="checkResult">The <see cref="ICheckResult"/> to convert.</param>
+    /// <returns>The <see cref="CheckResult"/> instance created from the given <see cref="ICheckResult"/>.</returns>
+    private static CheckResult CreateCheckResult(
+        ICheckResult checkResult)
     {
         CheckResult newCheckResult = new()
                                      {
-                                         FeedbackType = (CheckResult.Types.FeedbackType)((int)checkResult.GetFeedbackType() + 1),
-                                         Hidden = checkResult.IsHidden,
-                                         Score = checkResult.GetScore(),
-                                         FeedbackMessage = ResourceUtils.ResultToString(checkResult),
+                                         FeedbackType = (CheckResult.Types.FeedbackType.FeedbackCorrect),
+                                         Hidden = false,
+                                         FeedbackMessage = checkResult.FeedbackMessage
                                      };
-        foreach (Recognizers.Abstractions.ICheckResult childCheckResult in checkResult.GetChildFeedback())
+
+        switch (checkResult)
```

Comment:
Wordt ik nou gek of mist hier de LeafCheckResult? of komt hij daar niet? En wil je hier ook nog een default bij zetten? Of anders een Known ofz?

Reply:
Great thanks!